config.py

```diff
@@ -9,16 +9,27 @@ class Config(object):
     USE_SESSION_FOR_NEXT = True
     RESPONSE_OPERATIONS_UI_SECRET = os.getenv('RESPONSE_OPERATIONS_UI_SECRET', "secret")
     BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL')
+    SESSION_TYPE = "redis"
+    PERMANENT_SESSION_LIFETIME = os.getenv('PERMANENT_SESSION_LIFETIME', 43200)
+    REDIS_HOST = os.getenv('REDIS_SERVICE')
```

Comment:
Should this be REDIS_SERVICE rather than REDIS_HOST?

Reply:
Also, should we default this to something so it doesn't need to be set for every environment?

Reply:
Yes and yes

Reply:
The RAS/RM standard for now has been to *not* use defaults in base config classes, so that defaults aren't deployed to prod by mistake. Not saying I agree with that approach, but it's certainly the expected behaviour of services.